saltlint/rules/JinjaPillarGrainsGetFormatRule.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016, Will Thames and contributors
+# Copyright (c) 2018, Ansible Project
+# Modified work Copyright (c) 2019 Jeffrey Bouter
+
+from saltlint.linter import SaltLintRule
+import re
+
+
+class JinjaPillarGrainsGetFormatRule(SaltLintRule):
+    id = '211'
+    shortdesc = 'pillar.get or grains.get should be formatted differently'
+    description = "pillar.get and grains.get should always be formatted" \
+                  " like salt['pillar.get']('item') or grains['item1']"
+    severity = 'HIGH'
+    tags = ['formatting']
+    version_added = 'develop'
+
+    bracket_regex = re.compile(r"{{ ?(pillar|grains).get\(.+}}")
```

Comment:
The `{{` might also be followed by ` `, `-` and `+`.

Reply:
I have updated the regex. Please see whether you like this solution and resolve if you do.